easy_music_generator/tests/test_easy_music_generator.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+import unittest
+import easy_music_generator as emg
+
+
+class TestEasyMusicGenerator(unittest.TestCase):
+    """
+    Unit tests for easy_music_generator.py
+    """
+
+    def setUp(self):
```

Comment:
function naming convention should be lowercase separated by underscore. 
set_up